xarray/core/utils.py

```diff
@@ -431,3 +431,7 @@ def hashable(v):
     except TypeError:
         return False
     return True
+
+
+def not_implemented(*args, **kwargs):
+    return NotImplemented
```

Comment:
yep -- you can notice the red marker at the end of the line if the final new line is missing